### PR TITLE
Add plugin to evaluate limp

### PR DESCRIPTION
**Pull request by byxor**
Allows users to evaluate [limp](https://www.github.com/byxor/limp). 👌 

### Usage examples
```                                       
UncleBob: ".limp (+ 2 3)"
Chat_Bot: "5"
```                                    
```
UncleBob: ".limp (// 100 (- 5 2))"
Chat_Bot: "33"
```

:warning: Plugin comes with no tests, limp is already tested as its own python package.

### Notes
* Remember to **re-install pip dependencies** after merging.

### CODE REVIEW

File: steely/plugins/limp_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    def sendError(error):
+        send("Sorry, you have an error: {}".format(error))
+        
+    source_code = message
+    try:
+        result = limp.evaluate(source_code)
+    except SyntaxError as e:
+        sendError(e)
+    except NameError as e:
```
Comment (CianLR) [inline steely/plugins/limp_evaluation.py]: I think these could be 
```
except (S..., N...) as e:
```
but I'm not sure, I don't have a REPL to check

Comment (byxor) [inline steely/plugins/limp_evaluation.py]: That's a good point. I was going to make them respond differently but then decided it wasn't necessary.

File: steely/plugins/limp_evaluation.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+COMMAND = '.limp'
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+
+    def send(text):
+        bot.sendMessage(text, thread_id=thread_id, thread_type=thread_type)
+
+    def sendError(error):
```
Comment (sentriz) [inline steely/plugins/limp_evaluation.py]: can this boyo be snake_case pls

Comment (byxor) [inline steely/plugins/limp_evaluation.py]: Oops, the damn facebook api is throwing me off. Yes sir.

**Comment by sentriz** [comment]
gg

**Pull request merged**